students/tri_nguyen/lesson08/SparseArray_class.py

```diff
@@ -0,0 +1,45 @@
+class SparseArray:
+    def __init__(self, array):
+        self.array = array
+
+    def store_nonzero(self):
+        return {idx:item for idx, item in enumerate(self.array) if item !=0}
```

Comment:
this would be better to go into constructor and stored non-zero array into `self.array`